.github/workflows/ci-conan.yml

```diff
@@ -1,7 +1,7 @@
 name: libcosim CI Conan
 
 # This workflow is triggered on pushes to the repository.
-on: [push]
+on: [push, workflow_dispatch]
```

Comment:
Should enable [manual workflow trigger](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/)